paimon-flink/paimon-flink-cdc/src/main/java/org/apache/paimon/flink/action/cdc/CdcActionCommonUtils.java

```diff
@@ -140,8 +140,9 @@ public static String columnCaseConvertAndDuplicateCheck(
         if (caseSensitive) {
             return column;
         }
-        checkArgument(existedFields.add(column.toLowerCase()), columnDuplicateErrMsg.apply(column));
-        return column.toLowerCase();
```

Comment:
Here I think we can keep the filed name case in error message.

Reply:
is reserved, the variable used in the error message is `column`, and the returned result variable is `columnLowerCase`